src/components/notifications/Webhook.vue

```diff
@@ -13,60 +13,96 @@
 
     <div class="mb-3">
         <label for="webhook-content-type" class="form-label">{{
-            $t("Content Type")
+            $t("Request Body")
         }}</label>
         <select
             id="webhook-content-type"
             v-model="$parent.notification.webhookContentType"
             class="form-select"
             required
         >
-            <option value="json">application/json</option>
-            <option value="form-data">multipart/form-data</option>
+            <option value="json">Preset - application/json</option>
+            <option value="form-data">Preset - multipart/form-data</option>
+            <option value="custom">Custom Body</option>
```

Comment:
These options are not translatable. Is this intentional?

Reply:
Good point.